app.json

```diff
@@ -8,8 +8,7 @@
   ],
   "env": {
     "TOKEN": {
-      "description": "The Slack legacy token (generate on https://api.slack.com/custom-integrations/legacy-tokens).",
-      "required": true
```

Comment:
Need an explicit `"required": false` as the default is `"required": true`.

https://devcenter.heroku.com/articles/app-json-schema#env